modules/benchmark/R/load.data.R

```diff
@@ -16,6 +16,7 @@ load.data <- function(data.path, format, start_year = NA, end_year=NA, site=NA,
   require(PEcAn.benchmark)
   require(lubridate)
   require(udunits2)
+  require(dplyr)
```

Comment:
Ah, thanks. Don't know how I missed that. (Except that the whole system is confusing as hell...)